pages/linux/arithmetic.md

```diff
@@ -0,0 +1,16 @@
+# arithmetic
+
+> Quiz on simple arithmetic.
+> More information: <https://manpages.debian.org/bsdgames/arithmetic.6.en.html>.
+
+- Start an arithmetic quiz:
+
+`arithmetic`
+
+- Specify one or more arithmetic operation symbols to get problems on them:
```

Comment:
You can highlight argument in the example description to get it a meaning, like:
```suggestion
- Specify one or more arithmetic [o]peration symbols to get problems on them:
```

Reply:
That's quirky @navarroaxel , nice!